examples/secure-roks-cluster/README.md

```diff
@@ -1,89 +1,104 @@
-|private_endpoint|Add this option to connect to your Sysdig and logDNA service instance through the private service endpoint.|bool|true|No|
-|activity_tracker_instance_name|Name of Activity Tracker Instance. If `null` it creates an instance with `<var.resource_prefix>-at`|string|`null`|No|
-|custom_sg_rules|Custom Security rules. Find `custom_sg_rules` object table for more info. |list(object)|[]|No|
-|ip_ranges|Ordered List of ip_ranges on which subnets has to be created. If subnets are created in us-south region, elements in ip_ranges should be equivalent to respective ip range of [`us-south-1`, `us-south-2`, `us-south-3`]|list(string)||Yes|
+|create_timeout|Custom creation timeout for the cluster, in seconds.|string|N/A|No|
+|roks_kms_policy|Indicates if a Kubernetes Service to Key Protect service authorization policy exists in IAM. If false, a policy between the services is created.|bool|true|No|
+|kms_instance|Name of the Key Protect instance to use to encrypt the secrets in the cluster. If not provided, an instance is created with the following naming convention: `<var.resource_prefix>-kp`|string|N/A|No|
+|kms_key|Name of the root key in the Key Protect instance to use. If not provided, an instance is created with the following naming convention: `<var.resource_prefix>-kp-key`|string|N/A|No|
+|standard_key_type|Determines if the root key is a standard key or not. This variable is used only during creation of a Key Protect root key in this module.|bool|`false`|No|
+|sysdig_name| Name of the IBM Cloud Monitoring instance. If not provided, an instance is created with the following naming convention: `<var.resource_prefix>-sysdig`|string|N/A|No|
+|sysdig_access_key|The IBM Cloud Monitoring ingestion key that you want to use for your configuration.|string|N/A|No|
+|logDNA_name|Name of IBM Cloud Log Analysis instance. If not provided, an instance is created with the following naming convention: `<var.resource_prefix>-logdna`|string|N/A|No|
+|logdna_ingestion_key|The IBM Cloud Log Analysis ingestion key that you want to use for your configuration.|string|N/A|No|
+|private_endpoint|Add this option to connect to your Log Analysis and Monitoring service instances through the private cloud service endpoint.|bool|N/A|No|
+|activity_tracker_instance_name|Name of the IBM Cloud Activity Tracker instance. If not provided, a instance is created with the following naming convention: `<var.resource_prefix>-at`|string|`null`|No|
+|custom_sg_rules|Custom VPC security group rules. For more information, review the following [`custom_sg_rules` object](#custom_sg_rules-object). |list(object)|[]|No|
+|ip_ranges|An ordered list of IP address ranges on which the VPC subnets are created, for the region. If the subnets are created in the `us-south` region, the IP address ranges must match the IP address ranges of the region's zones, [`us-south-1`, `us-south-2`, `us-south-3`].|list(string)||Yes|
```

Comment:
I wasn't exactly sure what this meant? How do they know what the zone ranges are?

Reply:
Not sure exactly..
But users can refer to this https://cloud.ibm.com/docs/vpc?topic=vpc-vpc-addressing-plan-design&cm_mmc=IBMBluemixGarageMethod-_-MethodSite-_-10-19-15::12-31-18-_-vpc-addressing-plan-design